src/edge/edgeManager.ts

```diff
@@ -916,4 +956,72 @@ export class EdgeManager {
         const templates = this.get3rdPartyModuleTemplates();
         return templates ? templates.find((template) => template.name === name) : undefined;
     }
+
+    private async generateDevContainerDirectory(template: string, slnPath: string) {
+        const sourceContainersPath = this.context.asAbsolutePath(path.join(Constants.assetsFolder, Constants.containersFolder));
+        const sourceLibrayScriptsPath = this.context.asAbsolutePath(path.join(Constants.assetsFolder, Constants.libraryScriptsFolder));
+        let containerSource = "";
+        switch (template) {

```

Comment:
please add a comment to describe the incoming paths such that both cases are triggered